packages/browser-integration-tests/standalone-custom-key.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script>
+      !(function () {
+        var i = 'segment_analytics',
+          analytics = (window[i] = window[i] || [])
+        if (!analytics.initialize)
+          if (analytics.invoked)
+            window.console &&
+              console.error &&
+              console.error('Segment snippet included twice.')
+          else {
+            analytics.invoked = !0
+            analytics.methods = [
+              'trackSubmit',
+              'trackClick',
+              'trackLink',
+              'trackForm',
+              'pageview',
+              'identify',
+              'reset',
+              'group',
+              'track',
+              'ready',
+              'alias',
+              'debug',
+              'page',
+              'screen',
+              'once',
+              'off',
+              'on',
+              'addSourceMiddleware',
+              'addIntegrationMiddleware',
+              'setAnonymousId',
+              'addDestinationMiddleware',
+              'register',
+            ]
+            analytics.factory = function (e) {
+              return function () {
+                if (window[i].initialized)
+                  return window[i][e].apply(window[i], arguments)
+                var n = Array.prototype.slice.call(arguments)
+                if (
+                  [
+                    'track',
+                    'screen',
+                    'alias',
+                    'group',
+                    'page',
+                    'identify',
+                  ].indexOf(e) > -1
+                ) {
+                  var c = document.querySelector("link[rel='canonical']")
+                  n.push({
+                    __t: 'bpc',
+                    c: (c && c.getAttribute('href')) || void 0,
+                    p: location.pathname,
+                    u: location.href,
+                    s: location.search,
+                    t: document.title,
+                    r: document.referrer,
+                  })
+                }
+                n.unshift(e)
+                analytics.push(n)
+                return analytics
+              }
+            }
+            for (var n = 0; n < analytics.methods.length; n++) {
+              var key = analytics.methods[n]
+              analytics[key] = analytics.factory(key)
+            }
+            analytics.load = function (key, n) {
+              var t = document.createElement('script')
+              t.type = 'text/javascript'
+              t.async = !0
+              t.setAttribute('data-global-segment-analytics-key', i)
+              t.src =
+                'https://cdn.segment.com/analytics.js/v1/' +
+                key +
+                '/analytics.min.js'
```

Comment:
Is this testing the bundle that lives in this repo?

Reply:
Yes, after it's been built

Reply:
@zikaari I am still a bit confused -- to rephrase:

If this is loading analytics from the CDN, wouldn't this be currently testing against the version of analytics that doesn't have the changes reflected in this PR (i.e `if (globalAnalyticsKey)`)?

What am I missing?

Reply:
This does the redirection - https://github.com/segmentio/analytics-next/blob/d4f697502bcd38ec7a74c8545e354165bf41bec0/packages/browser-integration-tests/src/helpers/standalone-mock.ts#L18-L33

Reply:
ohhhh... I missed that detail! Thanks, that's excellent!